pyramid_sawing/main.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, handler, registry):
 
     def __call__(self, request):
         response = self.handler(request)
+        request.response = response
         self.logger.info(' ')
         return response
```

Comment:
Try returning `request.response` instead. That's likely the better thing to do. We probably don't want to pave over `request.response` just in case. Technically they should be the same but apparently you found they aren't. 😕

Reply:
The reason I'm suggesting this is just in case pyramid or another tween does something to the response we may be paving over.

Reply:
What I *think* is after `return response`, it probably gets assigned to `request.response`, but since we're trying to get the response **before** the return, we're not getting this locally stored response.

What do you think about just doing `request.response = self.handler(request)` and `return request.response`?